xml/System.Net.Http/HttpClientHandler.xml

```diff
@@ -344,7 +344,15 @@ After NuGet package v4.3.2, the default value of <xref:System.Net.DecompressionM
       <Docs>
         <summary>Gets the collection of security certificates that are associated with requests to the server.</summary>
         <value>The X509CertificateCollection that is presented to the server when performing certificate based client authentication.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+On .NET Core, the Key Usage attribute on the X509 certificate, if present, is required to have the value "Digital Signature" in order to be used when sending the request.
```

Comment:
nit: I'm not really sure if you did it uppercase specifically. If it is LGTM.

```suggestion
On .NET Core, the key usage attribute on the X509 certificate, if present, is required to have the value "Digital Signature" in order to be used when sending the request.
```

Reply:
I thought that it is capitalised because it is the name of the extension, but now I searched for it and it does not seem to be generally capitalised